include/tvSettings.h

```diff
@@ -283,7 +283,7 @@ tvError_t GetCurrentVideoFrameRate(tvVideoFrameRate_t *format);
  *
  * @pre TvInit() should be called before calling this API
  */
-tvError_t GetCurrentVideoSource(int *currentSource);
+tvError_t GetCurrentVideoSource(tvVideoSrcType_t *currentSource);
 
```

Comment:
APIs Removed
From include/tvSettings.h:
tvError_t GetTVSupportedHDR10Modes(tvDolbyMode_t *HDR10Modes[], unsigned short *count)
tvError_t GetTVSupportedHLGModes(tvDolbyMode_t *HLGModes[], unsigned short *count)
Replaced by a common API: GetTVSupportedDolbyVisionModes to retrieve supported modes for DolbyVision, HDR10, and HLG.

Are the above APIs are expected to replace in this phase 1?